File: Typescript/test/reverseDegree.test.ts
import { expect, test } from '@jest/globals';
import * as solution from '../src/reverseDegree';

test('Example 1', () => {
	let input = "abc";
	let expected = 148;
	expect(solution.reverseDegree(input)).toEqual(expected);
});

test('Example 2', () => {
	let input = "zaza";
	let expected = 160;
	expect(solution.reverseDegree(input)).toEqual(expected);
});

File: Dart/test/reverseDegree_test.dart
import 'package:flutter_test/flutter_test.dart';
import 'package:leetcode/reverseDegree.dart';

void main() {
  test("test example", () {
    var input0 = ["abc", "zaza"];
    var expected = [148, 160];

    final sln = Solution();
    for (var i = 0; i < input0.length; i++) {
      var output = sln.reverseDegree(input0[i]);
      expect(output, expected[i]);
    }
  });
}
